SMLHelper/Utility/SaveUtils.cs

```diff
@@ -12,5 +15,24 @@ public static string GetCurrentSaveDataDir()
         {
             return SaveLoadManager.temporarySavePath;
         }
+
+        /// <summary>
+        /// Registers a simple <see cref="Action"/> method to invoke whenever the player saves the game via the in game menu.
+        /// </summary>
+        /// <param name="onSaveAction">The method to invoke.</param>
+        public static void RegisterOnSaveEvent(Action onSaveAction)
```

Comment:
This should be moved to IngameMenuHandler

Reply:
Created `IngameMenuHandler` and moved the new methods to this space.